Makefile

```diff
@@ -32,6 +32,10 @@ PUBLIC_IP ?= 127.0.0.1
 CI_MINIKUBE_VERSION ?= v0.25.1
 CI_KUBECTL_VERSION ?= v1.9.4
 
+TRAVIS_IMAGE_VERSION ?= v0.1
+TEST_IMAGES = $(addprefix $(DOCKER_NAMESPACE)/, $(TEST_IMAGES_SUFFIX))
+TEST_IMAGES_SUFFIX = ffdl-lcm ffdl-trainer ffdl-metrics ffdl-databroker_s3 ffdl-ui ffdl-restapi ffdl-jobmonitor ffdl-controller log_collector ffdl-databroker_objectstorage tensorboard_extract
```

Comment:
This can probably be pulled from a config file? Makes it easier to extend.....

Reply:
Sure, I can put these into a config file.